Homework3/Codewars.py

```diff
@@ -0,0 +1,20 @@
+# Write a function called that takes a string of parentheses,
+# and determines if the order of the parentheses is valid.
+# The function should return true if the string is valid,
+# and false if it's invalid.
+# "()"              =>  true
+# ")(()))"          =>  false
+# "("               =>  false
+# "(())((()())())"  =>  true
+
+string = '(())((()())())'
+count = 0
+for char in string:
```

Comment:
more interesting task - we have 3 types of parentheses [{()}]

Reply:
done